packages/jest/src/setup.ts

```diff
@@ -100,6 +100,7 @@ export function setup(opts: Sa11yOpts = defaultSa11yOpts): void {
     autoCheckOpts.cleanupAfterEach ||= !!process.env.SA11Y_CLEANUP;
     if (process.env.SA11Y_AUTO_FILTER?.trim().length)
         autoCheckOpts.filesFilter ||= process.env.SA11Y_AUTO_FILTER.split(',');
+    if (useFilesToBeExempted().length != 0) autoCheckOpts.filesFilter ||= useFilesToBeExempted();
```

Comment:
I think you should concat the arrays and not `||`. With above code, if `autoCheckOpts.filesFilter` is truthy it won't consider the filters from `useFilesToBeExempted()`